### DISCUSSION TITLE
Edit Attributes directly from Product page

### DISCUSSION THREAD

**Discussion by grzegorzgit** [PrestaShop/PrestaShop#40178] [Ideas]
Hi,
I need to update names of our products' combinations. Unfortunately Prestashop is designed in a way that "combinations" (or attributes) are kinda separated from the product. So for example when I have a product in three sizes size L, size M, size X, and I need to change those names to eg. size L EU, size M EU, size X EU, then I have to know exactly where these attributes are in the attributes list.

We have plenty of products, all of them have combinations, this makes the attributes list very long. 70% of our products have package size, so I have plenty "package sizes" attributes generated in the attributes list. I need to update the combinations only in one product, just the name and leave everything as it is (reference, prices, even ID).

There is no way for me, to edit names of combinations on the product page, there is no indication by the combinations about their ID in the attributes list, so I cannot know which exactly attribute belongs to this product.

#### Describe the solution you'd like

The Name of the attribute (combination /  feature) should be editable on the product page.
Or there should be an indication on the product page, about its combinations IDs. 

#### Additional context

I personally find the way of attributes & features, and the combinations very unintuitive. I couldn't understand it at the beginning. During products import, there has to be done a lot of work preparing the CSVs. I know this is not easy, and every store will have different needs and different usage of it. But I think attributes should not be separated from product, this is an integral part of a product, a specific product.

**Comment by florine2623** [comment]
Hello @grezlikowski ,
Thanks for the suggestion.
Could you also share a screenshot of what you want it to look like in the future ?

I'll ping our product-team, please stay tuned.

**Comment by florine2623** [comment]
Ping @PrestaShop/product-team ,

What do you think of being able to edit attributes directly from the product page ?

Thanks!
